NuKeeper/NuGet/Process/NuGetUpdater.cs

```diff
@@ -14,10 +15,10 @@ public NuGetUpdater(IExternalProcess externalProcess = null)
             _externalProcess = externalProcess ?? new ExternalProcess();
         }
 
-        public async Task UpdatePackage(PackageUpdate update)
+        public async Task UpdatePackage(PackageIdentity newPackage, PackageInProject currentPackage)
```

Comment:
So a `PackageIdentity` is a type built into the nuget lib, consists of a package name and version. 
I'm thinking of changing this around even more...